doc/classes/Node.xml

```diff
@@ -129,21 +129,23 @@
 				    child_node.get_parent().remove_child(child_node)
 				add_child(child_node)
 				[/codeblock]
+				If you need the child node to be added below a specific node in the list of children, use [method add_child_below_node] instead of this method.
 				[b]Note:[/b] If you want a child to be persisted to a [PackedScene], you must set [member owner] in addition to calling [method add_child]. This is typically relevant for [url=https://godot.readthedocs.io/en/latest/tutorials/misc/running_code_in_the_editor.html]tool scripts[/url] and [url=https://godot.readthedocs.io/en/latest/tutorials/plugins/editor/index.html]editor plugins[/url]. If [method add_child] is called without setting [member owner], the newly added [Node] will not be visible in the scene tree, though it will be visible in the 2D/3D view.
 			</description>
 		</method>
 		<method name="add_child_below_node">
 			<return type="void">
 			</return>
-			<argument index="0" name="node" type="Node">
+			<argument index="0" name="appending_node" type="Node">
```

Comment:
I don't know if `appending_node` is clear enough, it doesn't sound immediately obvious to me. Maybe "sibling" would be better? Or "preceding_node".

Reply:
I was thinking the same thing, `appending_node` is almost as opaque as no name at all. `preceding_node` is immediately clear. `sibling` also isn't indicative of the purpose.

Reply:
Let's change it to `preceding_node ` then.